store/idp.go

```diff
@@ -261,6 +261,9 @@ func listIdentityProviders(ctx context.Context, tx *sql.Tx, find *FindIdentityPr
 	if err != nil {
 		return nil, FormatError(err)
 	}
+	if rows.Err() != nil {
```

Comment:
Put it after iteration L294

<img width="724" alt="image" src="https://user-images.githubusercontent.com/24653555/226110357-6cec0fe1-6d88-4ae0-9ab1-413730d9be50.png">